core/capabilities/integration_tests/mock_trigger.go

```diff
@@ -0,0 +1,113 @@
+package integration_tests
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"testing"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities"
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/datastreams"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+)
+
+const triggerID = "streams-trigger@1.0.0"
```

Comment:
I actually have a similar thing here, I'm hoping we could just make this a new trigger type useful for testing/development in the core node: https://github.com/smartcontractkit/chainlink/pull/13716/files#diff-905de3fd1acdb9e2daa319880d9a528cca7f034d3b8920eec76905b7e6b52acd